CHANGELOG.md

```diff
@@ -7,10 +7,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 [Unpublished]
 -------------
 ### Added
+- **New icons:** mruby(`.mrb`)
```

Comment:
Space between `mruby` and `(`, please. Once you add this, and remove line 15, I can merge this. :-)

I'll push an updated version of the icon-font shortly after we're finished.